spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'menu'
+
+describe Menu do
+
+  subject(:menu) { described_class.new }
+  let(:dishes) do
+    { "Prawn Crackers" => 1.51,
+      "Prawn Gyoza" => 3.51,
+      "Chicken Gyoza" => 3.11,
+      "Spring Rolls" => 3.15 }
+  end
+
+  let(:printed_menu) { "Prawn Crackers - £1.51, Prawn Gyoza - £3.51, Chicken Gyoza - £3.11, Spring Rolls - £3.15" }
+
```

Comment:
This is not quite the right use of `context`.  An place where it would work in this challenge would be something like:

```ruby
describe '#dish_on_menu?' do
  context 'when the dish is not on the menu' do
    it 'raises an error' do
      <test>
    end
  end
end
```